### DISCUSSION TITLE
Support for external stylesheets?

### DISCUSSION THREAD

**Discussion by rohitsathish** [zauberzeug/nicegui#484] [Q&A] [answered]
Is it possible to add an external stylesheet? Or is there any way (for eg., using a dict in a styles.py file) to manage the styling separately?

**Comment by falkoschindler** [comment, marked as answer]
You can use `ui.add_head_html(...)` to add arbitrary HTML to the head. This way you could link to a CSS file. See https://github.com/zauberzeug/nicegui/discussions/535#discussioncomment-5334932 for an example.

**Reply by rohitsathish** [reply]
Thank you! This works!

Is there any way to assign HTML classes to individual elements so that I can refer to those in the CSS stylesheet?

**Reply by falkoschindler** [reply]
Sure, you can use the `.classes()` method to assign any HTML class to an element.
Here is a similar discussion: https://github.com/zauberzeug/nicegui/discussions/1183
